openquake/server/static/js/engine_get_outputs.js

```diff
@@ -37,13 +37,6 @@
                 this.render();
             },
 
-            events: {
-                "click .btn-danger": "remove_calculation",
-                "click .btn-traceback": "show_traceback",
-                "click .btn-file": "on_run_risk_clicked",
-                "change .btn-file input": "on_run_risk_queued"
-            },
-
```

Comment:
These were leftovers from a cut/paste made from the calc list page to the outputs page